cmd/lncli/commands.go

```diff
@@ -1323,6 +1323,30 @@ func channelBalance(ctx *cli.Context) error {
 	return nil
 }
 
+var generateManPageCommand = cli.Command{
+	Name:   "generatemanpage",
+	Usage:  "Generates a man page for lncli.",
+	Hidden: true,
+	Action: actionDecorator(generateManPage),
+}
+
+func generateManPage(ctx *cli.Context) error {
+	docs, manErr := ctx.App.ToMan()
```

Comment:
Can just create and re-use a single `err` variable, it's easier to read if every error has the same name.